modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -2574,6 +2700,16 @@ such as `GOGC` and `GOMAXPROCS` env vars. Set these values at your own risk.
 | `string`
 | `clusterName` is the name of the cluster to appear in the flows data. This is useful in a multi-cluster context. When using {product-title}, leave empty to make it automatically determined.
 
+| `deduper`
+| `object`
+| `deduper` allows to sample or drop flows identified as duplicates, in order to save on resource usage.
+IMPORTANT: This feature is available as a Developer Preview.
```

Comment:
See previous Dev Preview concern.

Reply:
Globally, we've made mentions of Dev Preview render as "Unsupported" with an * that defines the unsupported nature of the feature. This is a unique challenge in sharing upstream/downstream autogenerated API docs, but I think this solution is a good compromise. Thanks for pointing this out in the PR.